### PR TITLE
Request to add another action command to flush applied filters

### PR DESCRIPTION
**Pull request by rgheleribefore**
Hello, your plugin is very useful but I had to made some adjusts to fit my needs. Thus, I added another command called 'flushSearchChip' to remove all filters already applied. It's very simple but saved my day and I was thinkink if it could be helpful to anyone else.

The way to easy is quite the same as 'addSearchChip'. See below: 

```
this.iframeElement = document.getElementById('iframe');
let message={actionType: "flushSearchChip""};
this.iframeWindow.postMessage(message, '*');
```

Thank you very much!

### CODE REVIEW

**Comment by ytzlax** [comment]
Hi rgheleribefore,
I glad to hear that the plugin is useful for you.
I want to merge your PR, but first please look at my request changes

File: public/hack.js [hunk 1]
```diff
@@ -127,6 +127,12 @@ uiModules.get('app/dashboard', []).run(function ($http, $location, kbnUrl, getAp
                 getAppState().save();
                 return;
 
+            case "flushSearchChip":
+                for(let i = 0; i < getAppState().filters.length; i++) {
```
Comment (ytzlax) [inline public/hack.js]: case "flushSearchChip":
                getAppState().filters = [];
                return;

File: public/hack.js [hunk 2]
```diff
@@ -127,6 +127,12 @@ uiModules.get('app/dashboard', []).run(function ($http, $location, kbnUrl, getAp
...
+                    getAppState().filters.pop();
+                }
```
Comment (ytzlax) [inline public/hack.js]: getAppState().filters = [];

File: public/hack.js [hunk 3]
```diff
@@ -127,6 +127,12 @@ uiModules.get('app/dashboard', []).run(function ($http, $location, kbnUrl, getAp
...
+                return;
```
Comment (ytzlax) [inline public/hack.js]: Instead the for loop simple add the next line:
`                getAppState().filters = [];
`

File: public/hack.js (hunk 2 above)
Comment (ytzlax) [inline public/hack.js]: Instead the for loop simple add the next line:
`                getAppState().filters = [];
`

**Comment by ytzlax** [comment]
I already did the change, what you think?
If its OK I will merge the PR

**Comment by rgheleribefore** [comment]
Great! It's work very well and you can merge if it's ok. As soon as I use more frequently the plugin, maybe I will have to suggest/implement new features to satisfy new cases. It's ok for you?

**Comment by ytzlax** [comment]
Hi,OK for me,
The last "to do" mission for me is to test this action on all kibana 5 versions, then I will merge it

**Comment by ytzlax** [comment]
Another thing, if you love this plugin, you can star it on the top of the screen

**Comment by rgheleribefore** [comment]
Ok. Star applied xD

**Comment by ytzlax** [comment]
I also update the wiki https://github.com/Webiks/kibana-API/wiki/Filter-functions

**Pull request merged**